index.html

```diff
@@ -845,6 +908,13 @@ <h3>
         <h3>
           Monitor available connection(s) and say hello
```

Comment:
Listening for incoming presentation connections

Reply:
Done